lib/features/splash/data/local/splash_repository_imp.dart

```diff
@@ -1,9 +1,13 @@
+import 'package:firebase_auth/firebase_auth.dart';
 import 'package:fiura_ecosystem/features/splash/domain/repositories/splash_repository.dart';
 
 class SplashRepositoryImp extends SplashRepository {
+  final FirebaseAuth auth = FirebaseAuth.instance;
```

Comment:
this should be provided on the constructor by the get it

Reply:
Fixed